Backend/app/rider-platform/rider-app/Main/spec/Storage/FrfsTicket.yaml

```diff
@@ -392,3 +393,26 @@ FRFSRecon:
     settlementAmount: (.amount)|I
     differenceAmount: Kernel.Prelude.fmap (.amount)|I
     currency: (Kernel.Prelude.Just . (.currency))|I
+
+FRFSConfig:
+  tableName: frfs_config
+  derives: "Generic,Show"
+  fields:
+    merchantOperatingCityId: Id MerchantOperatingCity
+    merchantId: Id Merchant
+    bookingStartTime: UTCTime
+    bookingEndTime: UTCTime
+    routndTripTicketLimit: Int
+    oneWayTicketLimit: Int
+    city: City
+
+  queries:
+    findByCity:
```

Comment:
You need to lookup based on `merchantOperatingCityId`

Reply:
yea i thought to do by that, but in frontend we don't get the `merchantOperatingCityId` at the place where we call our api, so we made based on `city`